event-handler/cli.go

```diff
@@ -55,6 +55,14 @@ type TeleportConfig struct {
 	// TeleportIdentityFile is a path to Teleport identity file
 	TeleportIdentityFile string `help:"Teleport identity file" type:"existingfile" name:"teleport-identity" env:"FDFWD_TELEPORT_IDENTITY"`
 
+	// TeleportRefreshEnabled will reload the identity file from disk on the
+	// configured interval.
+	TeleportRefreshEnabled bool `help:"Configures the identity file to be reloaded from disk at a configured interval." env:"FDFWD_TELEPORT_REFRESH_ENABLED"`
+
+	// TeleportRefreshInterval is how often the identity file should
+	// be reloaded from disk.
+	TeleportRefreshInterval time.Duration `help:"Configures how often the identity file should be reloaded from disk." env:"FDFWD_TELEPORT_REFRESH_INTERVAL" default:"1m"`
+
```

Comment:
We might want to expose those new flags in the helm chart
`charts/event-handler/templates/configmap.yaml`

Reply:
I was going to leave the helm chart stuff for a follow up PR that'll tie into the docs side of this